code/Model/SiteTree.php

```diff
@@ -1555,7 +1561,7 @@ protected function onBeforeWrite()
 
         // If Sort hasn't been set, make this page come after it's siblings
         if (!$this->Sort) {
-            $parentID = ($this->ParentID) ? $this->ParentID : 0;
+            $parentID = (int) ($this->ParentID ?: 0);
```

Comment:
Are you worried about a `null` database value for ParentID?

Don't need to both cast to int and use `?: 0`  Personally I'd remove the cast as `$this->ParentID ?: 0` is used elsewhere

Reply:
I guess I was worried about just whatever it would be.